doc/tutorials/pynest_tutorial/part_4_topologically_structured_networks.rst

```diff
@@ -378,6 +378,9 @@ Section 4.1 *Query functions*.
       'network_size': 20,
       'shape': (5, 4)}
 
+It may also be useful to look at the ``spatial`` property of the
+NodeCollection, which describes the layer properties. Other useful
+functions that may be of help are listed in NTUM Section 4.1.
```

Comment:
The term NTUM does not make sense anymore. It meant NEST Topology Users Manual.

Reply:
I think this was added by @stinebuu  as she had started working on the changes in the tutorials - I just wanted to mainly update links in this file.